packages/react/src/components/TextInput/TextInput.js

```diff
@@ -185,6 +193,11 @@ TextInput.propTypes = {
    * `true` to use the light version.
    */
   light: PropTypes.bool,
+
+  /**
+   * `true` to use fluid variation of input field.
+   */
+  fluidInput: PropTypes.bool,
```

Comment:
Do we know if forms with fluid inputs are supposed to be constructed? Is it an all-or-nothing type thing? (meaning all fluid versus not) or is it more mix-and-match?

Reply:
I think its all or nothing according this the issue at the very bottom is the example of the fluid form https://github.com/carbon-design-system/carbon/issues/5416

Reply:
Would it be helpful to set this at the form level then if it's all-or-nothing? Could help to avoid repeating `fluidInput` repeatedly and accidentally mixing-and-matching. For example:

```jsx
<Form fluid>
  <TextInput />
</Form>

// Or
<FluidForm>
  <TextInput />
</FluidForm>
```

Reply:
I believe the fluid inputs can be used outside the context of a Form. We could always add Form level controls as an enhancement in the future, so you don't have to toggle it on for each form item.

Reply:
Ah got it, that makes sense. Is fluid basically like another size that folks can choose from basically?

Reply:
Nevermind! According to Design it should only be used in the context of a Form or Title. Since that is the case, it would make more sense to add a prop to the Form and Tile components to handle toggling this feature on / off